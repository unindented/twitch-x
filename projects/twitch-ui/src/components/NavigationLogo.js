import React from 'react'
import styled from 'styled-components'
import glitch from 'twitch-assets/build/web/icons/glitch.svg'
import NavigationItem from './NavigationItem'

const NavigationLogoRoot = styled.h1`
  margin: 0;
  background-color: ${props => props.theme.colors.secondary};
`

export default function NavigationLogo () {
  return (
    <NavigationLogoRoot>
      <NavigationItem
        icon={glitch}
        iconAlt='Twitch'
      />
    </NavigationLogoRoot>
  )
}
