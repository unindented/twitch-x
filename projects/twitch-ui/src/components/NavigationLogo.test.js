import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import NavigationLogo from './NavigationLogo'

describe('NavigationLogo', () => {
  let tree

  beforeEach(() => {
    const renderer = new ShallowRenderer()
    renderer.render(<NavigationLogo />)
    tree = renderer.getRenderOutput()
  })

  it('renders', () => {
    expect(tree).toMatchSnapshot()
  })
})
