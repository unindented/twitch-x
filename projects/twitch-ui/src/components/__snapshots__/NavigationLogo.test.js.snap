// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLogo renders 1`] = `
.c2 {
  display: block;
  width: 50px;
  height: 50px;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding: 10px;
}

.c3 {
  display: block;
  width: 100%;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  margin: 0;
  background-color: #4b367c;
}

<h1
  className="c0"
>
  <div
    className="c1"
  >
    <span
      className="c2"
    >
      <img
        alt="Twitch"
        className="c3"
        src="file-mock"
      />
    </span>
  </div>
</h1>
`;
